app/bundles/ReportBundle/Views/FormTheme/Report/table_order_row.html.twig

```diff
@@ -0,0 +1,14 @@
+{% set labelClass = from.vars.label_attr.class is not empty ?: 'control-label' %}
+{% block table_order_row %}
```

Comment:
Custom form themes should have `_` in front of the block name. ie `_table_order_row` or it will never be used.

---

I've also noticed that the template names aren't always a one to one match with the fields they are for. You can use the profiler to find the `block_prefixes` (see screenshot) you need to use for the block names here. Also take a look at https://symfony.com/doc/current/form/form_themes.html#form-fragment-naming So if the Block Prefix is `_report_tableOrder` and you want to only modify the label, the block name is `_report_tableOrder_label`.

![Screenshot from 2023-02-06 12-20-03](https://user-images.githubusercontent.com/447419/217040598-d5502357-49ba-4566-966c-2f2a50544dbf.png)